admin/module/catalog-admin-module-sources/src/main/webapp/js/view/Source.view.js

```diff
@@ -123,18 +140,8 @@ function (ich,Marionette,_,$,Q,ModalSource,EmptyView,Service,Status,wreqr,Utils,
             evt.stopPropagation();
         },
         updateStatus: function(statusList) {
-            var model = this.model;
-            var currentStatus = statusList.find(function(status) {
-                return status.id === model.id;
-            });
-            var available = model.get('available');
-            if (currentStatus) {
-                var curAvail = currentStatus.get('available');
-                if (available !== curAvail) {
-                    model.set('available', curAvail);
-                    this.render();
-                }
-            }
+            this.model.set('available', statusList.models[0].get('value'));
```

Comment:
Use the at function instead of doing models[0].

http://backbonejs.org/#Collection-at